lib/daru/index/index.rb

```diff
@@ -44,8 +44,9 @@ def each(&block)
     end
 
     attr_reader :relation_hash, :size
+    attr_accessor :name
 
-    def initialize index
+    def initialize index, opts={} # rubocop:disable Metrics/MethodLength
```

Comment:
Yep. (But not just to blindly follow the cop -- this suggestion will make the code better, I believe)